blazor/multicolumn-combobox/filtering.md

```diff
@@ -0,0 +1,135 @@
+platform: Blazor
+control: MultiColumn ComboBox
+documentation: ug
+---
+
+# Filtering in Blazor MultiColumn ComboBox Component
+
+The MultiColumn ComboBox offers built-in functionality for filtering data items when the [AllowFiltering]() option is enabled. The filtering action starts as soon as you begin typing in the search box. By default, the AllowFiltering property is set to `false`.
+
+## Local data
+
+The following code demonstrates the filtering functionality with local data in the MultiColumn ComboBox component.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/local-data.razor %}
+
+{% endhighlight %}
+
+![Blazor MultiColumn ComboBox with local data filtering](./images/filtering/blazor_combobox_local-data.png)
+
+## Remote data
+
+For remote data, every key press and filter action request is processed on the server side.
+
+The following code illustrates the filtering capabilities using the [ODataAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odata-adaptor) in the MultiColumn ComboBox component, utilizing the [Query](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Data.Query.html) property.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/remote-data.razor %}
+
+{% endhighlight %}
+
+## Filter type
+
+You can use [FilterType]() property to specify on which filter type needed to be considered on the search action of the component. The available `FilterType` and its supported data types are:
+
+FilterType     | Description
+------------ | -------------
+  [StartsWith]()       | Checks whether a value begins with the specified value.
+  [EndsWith]()     | Checks whether a value ends with specified value.
+  [Contains]()      | Checks whether a value contained with specified value.
+
+In the following example, `EndsWith` filter type has been mapped to the `FilterType` property.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/filter-type.razor %}
+
+{% endhighlight %}
+
+![Blazor MultiColumn ComboBox with Filter Type](./images/filtering/blazor_combobox_filter-type.png)
+
+## Minimum filter length
+
+When filtering list items, you can specify a character count limit to trigger a remote request and retrieve filtered data for the DropDownList. This can be achieved through manual validation using the [Filtering event arguments]() within the [Filtering]() event handler.
+
+In the following example, the remote request does not fetch the search data until the search key contains three characters.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/filtering/minimum-filter-length.razor %}
+
+{% endhighlight %}
+
+![Blazor MultiColumn ComboBox with Minimum filter length](./images/filtering/blazor_combobox_minimum-filter-length.gif)
+
+## Multi column filtering
+
+To enable multi-column support in the built-in Syncfusion Blazor theme files, simply add the `e-multi-column` class to the [CssClass]() property.
```

Comment:
This is not required here, if the multiple field filtering using predicates is explained in custom filtering section